### ISSUE TITLE
Brush texture - grey box instead of a texture

### ISSUE THREAD

**Issue by sighmon** [adamwulf/JotUI#28]
Hi there,

Thanks so much for your project. I'm trying to use it for an exhibition piece where visitors can draw their pain (in response to some simulated and generated short electric shocks), and wanted to add a textured brush to the app.

The brush image I'm trying to use is:

<img src="https://github.com/MODatUniSA/mod-draw-your-pain/raw/master/JotUI/JotUI/charcoal.png">

And to test it out I've jammed it into the ```JotViewDelegate```:

```ObjectiveC
#pragma mark - JotViewDelegate

- (JotBrushTexture*)textureForStroke {
    if (textureActive) {
        return [JotHighlighterBrushTexture sharedInstance];
    } else {
        return [[self activePen] textureForStroke];
    }
}
```

Setting the JotHighlighterBrushTexture to my `charcoal.png` image:

```ObjectiveC
//        UIImage* highlighter = [UIImage imageWithContentsOfFile:[bundle pathForResource:@"highlighter" ofType:@"png"]];
        UIImage* highlighter = [UIImage imageWithContentsOfFile:[bundle pathForResource:@"charcoal" ofType:@"png"]];
```

My code is here if you want to look through it/run it: https://github.com/MODatUniSA/mod-draw-your-pain

The result is just a semi-transparent square that doesn't respond to colour changes anymore.

<img src="https://lh4.googleusercontent.com/K4IK_JFsTGxmIwiDbJQ56VaCicDig2KlMIXhb0li-g_5Miexnk6GBaoliA8oUQy1yIWJzU_c2U8-TMv7SGmq=w2848-h1632">

The final interface is black, so I'd like the brush texture to be white:

<img src="https://lh3.googleusercontent.com/MjDCMaVAETaMu9qtls6aze0GDV4weg21KZUCRT1K7XTpaJjISXvLb7YX3balChR8IFBB6toLvpU43TATBqV3=w2848-h1632">

Any ideas?
Best, Simon.

**Comment by adamwulf** [comment]
I downloaded the code and found 2 issues that seemed to be causing the trouble:

first - the texture needs to be white instead of black. the shader i use to render the strokes uses that color value from the png in coordination w/ the user's chosen color. Here's the texture I used that did work:

![charcoal](https://user-images.githubusercontent.com/251087/36997367-12f9a420-207f-11e8-99fa-63ffaad263c4.png)

The other thing I did was to add charcoal.png to the modpain app's "Copy Bundle Resources" build phase. I think I actually need to update the main JotUI project to add the texture files to jotuiexample as well, as this step didn't use to be necessary.
